Documentation/gettingstarted/hubble-install.rst

```diff
@@ -0,0 +1,47 @@
+.. tabs::
+
+   .. group-tab:: Linux
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -LO https://github.com/cilium/hubble/releases/latest/download/hubble-linux-amd64.tar.gz
+         curl -LO https://github.com/cilium/hubble/releases/latest/download/hubble-linux-amd64.tar.gz.sha256sum
+         sha256sum --check hubble-linux-amd64.tar.gz.sha256sum
+         tar zxf hubble-linux-amd64.tar.gz
+
+      and move the ``hubble`` CLI to a directory listed in the ``$PATH`` environment variable. For example:
+
+      .. parsed-literal::
+
+         sudo mv hubble /usr/local/bin
+
+   .. group-tab:: MacOS
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -LO https://github.com/cilium/hubble/releases/latest/download/hubble-darwin-amd64.tar.gz
+         curl -LO https://github.com/cilium/hubble/releases/latest/download/hubble-darwin-amd64.tar.gz.sha256sum
+         sha256sum --check hubble-darwin-amd64.tar.gz.sha256sum
+         tar zxf hubble-darwin-amd64.tar.gz
+
+      and move the ``hubble`` CLI to a directory listed in the ``$PATH`` environment variable. For example:
+
+      .. parsed-literal::
+
+         sudo mv hubble /usr/local/bin
+
+   .. group-tab:: Windows
+
+      Download the latest hubble release:
+
+      .. parsed-literal::
+
+         curl -L https://github.com/cilium/hubble/releases/latest/download/hubble-windows-amd64.tar.gz
+
+      and move the ``hubble.exe`` CLI to a directory listed in the ``%PATH%`` environment variable after
+      extracting it from the tarball.
```

Comment:
I think the Windows instructions are lacking as the tarball is never extracted. By the way, is `curl` available on Windows? How does one download a file, verify the SHA256 checksum and extract a `.tar.gz` file natively on Windows?

Reply:
thanks @Rolinh this is super-helpful. here is the updated doc: https://5ee2681c83fec8187682a6e9--cilium-docs.netlify.app/gettingstarted/k8s-install-gke.html#enable-hubble